content/en/docs/js/_index.md

```diff
@@ -23,6 +28,5 @@ You can find release information [here](https://github.com/open-telemetry/opente
 ## Further Reading
 
 - [OpenTelemetry for JavaScript on GitHub](https://github.com/open-telemetry/opentelemetry-js)
-- [Getting Started](getting_started)
```

Comment:
**Getting started** shouldn't be under **Further reading**, but it is better there than not at all. I'm not sure why this was removed from the otel-js/website_docs version. Anyhow, I propose that we add it back. Any objections @dyladan or @austinlparker?

Reply:
Might have been removed because it's a link to a folder in otel-js? https://github.com/open-telemetry/opentelemetry-js/tree/main/getting-started

I say we re-add it as a link to that folder.

Reply:
Ok, I'll add it back for now (even if a **Getting started** page shouldn't be considered **Further reading**), at least until we rework the landing pages.

Reply:
Done